### PR TITLE
this is a Dublin guide

### PR DESCRIPTION
**Pull request by igor101**
Check here for places to eat in Dublin

### CODE REVIEW

**Comment by ntagg** [comment]
Always wanted to go to Dublin!

File: dublin-guide.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Places to eat in Dublin
+
+# The White Moose Cafe
```
Comment (maxxrafen) [inline dublin-guide.md:3]: What's the best thing to order?

Comment (igor101) [inline dublin-guide.md:3]: Full Irish breakfast. And be nice to the owner, he's vindictive, but entertaining, on his Facebook page.

Comment (sethuprem) [inline dublin-guide.md:3]: interesting

**Comment by maxxrafen** [comment]
Looks Good! :+1:

**Pull request merged**